### PR TITLE
Tutorial for Biometric Liveness Verification

### PR DESCRIPTION
**Pull request by romanstrobl**

### CODE REVIEW

File: docs/tutorials/Biometric-Liveness-Verification/Readme.md [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+User Data Store:
+```shell
+docker pull wultra.jfrog.io/wultra-docker/user-data-store:1.3.0-a2871e2f8d033dd4d00bf1a39456fc503be79194
+```
+
+```shell
+docker pull wultra.jfrog.io/wultra-docker/liveness-check-proxy:0.1.0-SNAPSHOT-2024.06.26-0f5031f9d5ca260aa40c28305ddb2e93a7f4e74e
+```
+
+TODO: use the release version of Docker image
```
Comment (banterCZ) [inline docs/tutorials/Biometric-Liveness-Verification/Readme.md]: Will we merge this before the Liveness Check Proxy release?

**Review by banterCZ — APPROVED** [review]
Good job!

File: docs/tutorials/Biometric-Liveness-Verification/Readme.md (hunk 1 above)
Comment (romanstrobl) [reply docs/tutorials/Biometric-Liveness-Verification/Readme.md]: I will wait for the Liveness Check Proxy release.

File: docs/tutorials/Biometric-Liveness-Verification/Readme.md [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+
+<!-- AUTHOR romanstrobl 2024-09-26T00:00:00Z -->
+<!-- SIDEBAR _Sidebar.md sticky -->
+<!-- TEMPLATE tutorial -->
+
+This tutorial explains how Wultra's Biometric Liveness Verification components can be deployed, configured and integrated into your application. This solution enables to verify users using advanced biometric verification techniques against their static photos, such as passport photos.
+
+## Introduction
+
+Wultra's Biometric Liveness Verification solution consists of two components which require to be deployed. The components are available as Docker images in Wultra JFrog Artifactory.
```
Comment (zcgandcomp) [inline docs/tutorials/Biometric-Liveness-Verification/Readme.md]: We require mobile SDK to be used as well.

File: docs/tutorials/Biometric-Liveness-Verification/Readme.md [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+```
+
+```shell
+docker pull wultra.jfrog.io/wultra-docker/liveness-check-proxy:0.1.0-SNAPSHOT-2024.06.26-0f5031f9d5ca260aa40c28305ddb2e93a7f4e74e
+```
+
+TODO: use the release version of Docker image
+
+You can use one of the supported databases:
+- PostgreSQL 9.5.4 or newer
```
Comment (zcgandcomp) [inline docs/tutorials/Biometric-Liveness-Verification/Readme.md]: PostgreSQL 13 is our target see https://www.postgresql.org/support/versioning/

File: docs/tutorials/Biometric-Liveness-Verification/Readme.md [hunk 4]
```diff
@@ -0,0 +1,364 @@
...
+- Oracle Database 11g, 12c, 19c, or 21c
```
Comment (zcgandcomp) [inline docs/tutorials/Biometric-Liveness-Verification/Readme.md]: 19c - we support long term releases still supported with Extended Support / Unlimited Licence Agreement
https://support.oracle.com/knowledge/Oracle%20Database%20Products/742060_1.html

File: docs/tutorials/Biometric-Liveness-Verification/Readme.md [hunk 5]
```diff
@@ -0,0 +1,364 @@
...
+<!-- AUTHOR romanstrobl 2024-09-26T00:00:00Z -->
+<!-- SIDEBAR _Sidebar.md sticky -->
+<!-- TEMPLATE tutorial -->
+
+This tutorial explains how Wultra's Biometric Liveness Verification components can be deployed, configured and integrated into your application. This solution enables to verify users using advanced biometric verification techniques against their static photos, such as passport photos.
+
+## Introduction
+
+Wultra's Biometric Liveness Verification solution consists of two components which require to be deployed. The components are available as Docker images in Wultra JFrog Artifactory.
+
```
Comment (zcgandcomp) [inline docs/tutorials/Biometric-Liveness-Verification/Readme.md:28]: Some high-level explanation or picture would be great. To understand the whole concept.

**Review by zcgandcomp — CHANGES_REQUESTED** [review]
It's a good start; check the prerequisites. We need to add an overview and mobile part to have an end-to-end guideline.

File: docs/tutorials/Biometric-Liveness-Verification/Readme.md (hunk 5 above)
Comment (romanstrobl) [reply docs/tutorials/Biometric-Liveness-Verification/Readme.md:28]: Ok, I will write an explanation.

**Pull request merged**